### PR TITLE
config: describe parallel parameters

### PR DESCRIPTION
**Pull request by staabm**
.. for better DX

### CODE REVIEW

File: packages/Config/RectorConfig.php [hunk 1]
```diff
@@ -68,6 +68,13 @@ public function enableCollectors(): void
         SimpleParameterProvider::setParameter(Option::COLLECTORS, true);
     }
 
+    /**
+     * Enables parallel processing utilizing several worker processes.
+     *
+     * @param positive-int $seconds timeout in seconds, after a worker will be force-killed.
+     * @param positive-int $maxNumberOfProcess number of parallel worker processes. The higher the number, the more overall memory is consumed.
+     * @param positive-int $jobSize number of files a worker processes at once. A higher number means faster processing but also more peak memory consumption.
```
Comment (TomasVotruba) [inline packages/Config/RectorConfig.php:76]: These might be missleading, as based on the desscription, I'd pick 1 and 1 for the process and job size :)
The lower memory might be case, but also faster process. This depends very on specific project and machine.

I'd keep this on the user experience and personal preference.

**Comment by TomasVotruba** [comment]
I'd leave this on the user and their own experience, as we'd have to write full documentation about parallel process in general to make it accurate :) 

Thanks but I want to keep configs simple

File: packages/Config/RectorConfig.php (hunk 1 above)
Comment (staabm) [reply packages/Config/RectorConfig.php:76]: so should I drop the hints regarding memory/speed, or should we close it alltogether?

Comment (staabm) [reply packages/Config/RectorConfig.php:76]: btw: even in projects where one of the rector experts is contributing on a rugular basis we had non-sensical settings:

https://github.com/codeigniter4/CodeIgniter4/pull/7918/files#r1321617415

I am not sure its a good idea to let people shoot into their own foot

Comment (TomasVotruba) [reply packages/Config/RectorConfig.php:76]: Github PR are lagging, so only now I see your comments.

That's anecdotical evidence tailored to the exact project itself, I've had a good experience on other projects so far. It depends :) 
This topic should be educated standalone, regardless Rector configs.

**Pull request closed without merging**